<?xml version="1.0"?>
<launch>   
    <!-- send urdf to param server -->
    <arg name="scenario_parameters" value="sim:=false"/> 
    <param name="sensor_description" command="$(find xacro)/xacro --inorder '$(find ati_force_torque)/description/urdf/sensor_mini58.urdf.xacro' $(arg scenario_parameters)"/>  
    
    <!-- sensor state publisher -->
    <node ns="fts" pkg="robot_state_publisher" type="robot_state_publisher" name="sensor_state_publisher">
        <param name="publish_frequency" type="double" value="200.0" />
        <param name="tf_prefix" type="string" value="" />
        <remap from="robot_description" to="sensor_description" />
    </node>
    
    <include ns="fts" file="$(find ati_force_torque)/launch/sensor_parameters_rs485.launch"/>

    <!-- start FTS Driver Node -->
    <node pkg="force_torque_sensor" type="force_torque_sensor_node" name="fts" ns="fts" cwd="node" respawn="false" output="screen">
    </node>

</launch>
